tsconfig.json

```diff
@@ -14,6 +14,7 @@
         "noEmitOnError": true,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
+        "experimentalDecorators": true,
```

Comment:
Can remove these, largely just scaffolding, using the decorators to ensure an impl class matches an xxxCons type

Reply:
yep, will do

Reply:
Removed the decorators